Assets/Shopify/UIToolkit/Components/ProductCache.cs

```diff
@@ -0,0 +1,46 @@
+namespace Shopify.UIToolkit {
+    using System;
+    using System.Collections.Generic;
+    using Shopify.Unity;
+
+    public class ProductCache {
+        private string _cursor = null;
+        private bool _complete = false;
+
+        private List<Product> _products;
+
+        public ProductCache() {
+            _products = new List<Product> ();
+        }
+
+        public string Cursor {
```

Comment:
you can make these properties more concise by using `public string Cursor { get; private set; }`

Reply:
When declared like this, I take it `private string _cursor = null;` would not be required?